GAE/war/WEB-INF/web.xml

```diff
@@ -451,17 +462,23 @@
 		<url-pattern>/databackout</url-pattern>
 	</filter-mapping>
 	<filter-mapping>
-    <filter-name>restAuthFilter</filter-name>
-    <url-pattern>/surveyedlocale</url-pattern>
-</filter-mapping>
+        <filter-name>restAuthFilter</filter-name>
+        <url-pattern>/surveyedlocale</url-pattern>
+    </filter-mapping>
 	<filter-mapping>
 		<filter-name>restAuthFilter</filter-name>
 		<url-pattern>/surveyrestapi</url-pattern>
 	</filter-mapping>
-	<filter-mapping>
-		<filter-name>restAuthFilter</filter-name>
-		<url-pattern>/rawdatarestapi</url-pattern>
-	</filter-mapping>
+    <filter-mapping>
+        <filter-name>restAuthFilter</filter-name>
+        <url-pattern>/rawdatarestapi</url-pattern>
+    </filter-mapping>
+    <!-- Experiment. Also protected by Spring. -->
+    <filter-mapping>
+        <filter-name>restAuthFilter</filter-name>
+        <url-pattern>/rest/reports</url-pattern>
```

Comment:
Spring already secures this end point so the restAuthFilter should actually be for securing the `/app_worker/reports` endpoint

Reply:
Yes, but the report engine cannot use Spring auth

Reply:
I suggest that since you already created the `ReportServlet` to run the task, the report engine can also use it to update when reports are generated.

Reply:
I had been hoping to keep the complexity down. Especially since it looked like we were redoing auth soon anyway. But ok. Changing the API again.